docs/spelling-wordlist.txt

```diff
@@ -18,6 +18,7 @@ implementers
 linter
 linters
 positionally
+perf
```

Comment:
I mean `perf` isn't really a word -- do you see where it's creeping in in the docs? Possibly somewhere should enclose it in ` (which would make this go away)?

Reply:
I havent myself included perf anywhere in the docs yet, ig when its building the docs and getting the supported commands it would be adding perf right ? That is what it raising the spelling error to handle which I added the keyword in our wordlist.

Reply:
Oh, that's odd, I guess that's a minor bug in `sphinx-click`'s output that hasn't been obvious given all the other command names are words... I mean this is pretty tiny obviously but perhaps put this instead in `docs/cli.rst` just above the click directive:

```
.. spelling:word-list::
   perf
```

so that at least it only applies in that file?